devicon.json

```diff
@@ -3117,8 +3457,29 @@
         "color": "#7C6DB2",
         "aliases": []
     },
+    {
+        "name": "knockout",
+        "altnames": [
+            "knockoutjs"
+        ],
+        "tags": [
+            "framework",
+            "javascript"
+        ],
+        "versions": {
+            "svg": [
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain-wordmark"
+            ]
+        },
+        "color": "#e42e16",
+        "aliases": []
+    },
```

Comment:
Duplicate entry of `knockout`

Reply:
6e9a895871f9c489551e347f4b9c05ee0830f8ae